api/src/test/java/org/pmiops/workbench/utils/mappers/WorkspaceMapperTest.java

```diff
@@ -174,10 +173,8 @@ public void testConvertsDbToApiWorkspace() {
   public void testConvertsFirecloudResponseToApiResponse() {
     final WorkspaceResponse resp =
         workspaceMapper.toApiWorkspaceResponse(
-            sourceDbWorkspace,
```

Comment:
this was a specialized version only for this test.  removed to reduce the number of different versions